README.md

```diff
@@ -519,7 +519,7 @@ The following are the predefined `runMode` configurations for convenience. They
 |Predefined runMode|Description|JestRunMode|
 |:---:|---|---|
 |"watch"| run tests by watchman | {type: "watch", revealOutput: "on-run"} |
```

Comment:
Hi, thanks for bringing this to our attention. It turns out this section has even more outdated information😓: there is no longer a `revealOutput` attribute. Would you mind removing all the revalOutput as well while you are at it? Thanks!

Reply:
@connectdotz 
Thank you for the review, I have deleted it!

[docs: delete revealOutput from README.md](https://github.com/jest-community/vscode-jest/pull/1126/commits/7b897a28d14341cf3fe17425a127f022f543f6ff)